### ISSUE TITLE
[Bug] Works only on CPU, not using GPU

### ISSUE THREAD

**Issue by Noctara** [lkwq007/stablediffusion-infinity#201]
I give up... I tried to install it in all ways that i can found here for 4 days, but it always ends with GPU not used for generation. I have 1060 6gb and other variants of Stable Difusion works well for me, but not Infinity... Any help, please?

**Comment by TheBest6337** [comment]
Yeah same with mine. I have an RTX2060 and it uses none GPU and only my CPU

**Comment by 51M0N3-3XC** [comment]
for me too, same behavior, and it's a pity (tesla P40 on windows)
I lost half a day testing but nothing, not even an error, it only uses the cpu
Does anyone have an idea what's going wrong?
